tests/test_client.py

```diff
@@ -145,6 +145,23 @@ def test_client_upsert_user():
     assert (
         http_client_testing.received_request.__str__() == 'HttpRequest(https://api.journy.io/users/upsert, Method.POST, {"content-type": "application/json", "user-agent": "python-sdk/0.0.0", "x-api-key": "api-key"}, {"identification": {"email": "user@journy.io", "userId": "user_id"}, "properties": {"hasdog": false, "name": "Manu"}})')
 
+def test_client_delete_user():
+    http_client_testing = HttpClientTesting(created_response)
```

Comment:
Maybe add test with Status code 202: `Accepted`? Might be confusing to others :wink: